shotover-proxy/src/transforms/util/mod.rs

```diff
@@ -12,7 +12,7 @@ pub mod unordered_cluster_connection_pool;
 pub struct Request {
     pub messages: Message,
     pub return_chan: Option<tokio::sync::oneshot::Sender<Response>>,
-    pub message_id: Option<u16>,
+    pub message_id: Option<i16>,
```

Comment:
can we have negative message ids?

Reply:
I asked about that here: https://github.com/shotover/shotover-proxy/pull/356#discussion_r756509944

Reply:
Presumably its "because java" but I havent investigated why cassandra-protocol changed it.